test/cfa/ntp_conf_test.rb

```diff
@@ -0,0 +1,365 @@
+require_relative "../test_helper"
+require "cfa/memory_file"
+require "cfa/ntp_conf"
+
+def ntp_disk_file
+  path = File.expand_path("../../fixtures/cfa/ntp.conf", __FILE__)
+  File.read(path)
+end
+
+describe CFA::NtpConf do
+  subject(:ntp) { described_class.new(file_handler: ntp_file) }
+
+  let(:ntp_file) { CFA::MemoryFile.new(content) }
+
+  before do
+    ntp.load
+  end
+
+  context "when ask for the collection" do
+    context "#servers" do
```

Comment:
Although [#describe](https://github.com/rspec/rspec-core/blob/master/lib/rspec/core/example_group.rb#L272) and [#context](https://github.com/rspec/rspec-core/blob/master/lib/rspec/core/example_group.rb#L277) just aliases, I rather prefer using `describe` in this case. You're not setting the context but describing the method.